packages/shared/src/storage/GoogleDriveStorage/GoogleDriveStorage.ts

```
@@ -0,0 +1,357 @@
+    query?: string;
+    pageSize?: number;
+  }): Promise<{ files: IGoogleDriveFile[] }> {
+    this.checkIOSNotSupported();
+    this.checkAndroidSupported();
+
+    // Ensure user is signed in
+    const signedIn = await GoogleSignin.isSignedIn();
+    if (!signedIn) {
+      throw new OneKeyLocalError(
+        'Not signed in to Google. Please sign in first.',
+      );
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)